docs/sequence_diagrams/notification-schedulerFlow.plantuml

```diff
@@ -0,0 +1,57 @@
+@startuml
```

Comment:
Please, change what is applicable from above comments.

Reply:
Was done as part of the above requests.